app/assets/javascripts/app/controllers/_channel/twitter.coffee

```diff
@@ -268,6 +268,10 @@ class AccountEdit extends App.ControllerModal
           position += 1
       else
         search.push params.search
+    if params.track_retweets
+      params.track_retweets = 1
```

Comment:
I was about to do that but putting booleans there makes Zammad save them as string in the database. Can you tell me if they get converted back to booleans when I access the channel in the backend?